src/management/endpoint.c

```diff
@@ -157,6 +161,25 @@ static int endpoint_create_process_data(struct controller_api *endpoint_create)
 			 endpoint_create_api->req.endpoint_name,
 			 endpoint_name_len);
 
+	rc = cfs_fs_lookup(&endpoint_name, &fs);
+        if (rc != 0) {
+                log_err("Can't create endpoint for non existent fs");
+                rc = -ENOENT;
+		request_set_errcode(request, rc);
+		endpoint_create_send_response(endpoint_create, NULL);
+                goto error;
+        }
+ 
+        /* get filesyetm ID */
+        cfs_fs_get_id(fs, &fs_id);
```

Comment:
Question: The process of doing a lookup for FS via its name and then fetching FS id, is it good to do it here inside rest - request handling code. Any specific reasons for doing it here? Or will it be better to do it in CFS APIs. 
@sachinpunadikar , @SwapnilGaonkar7  What are your thoughts on this?

Reply:
As long as cfs_fs_lookup and cfs_fs_get_id are published API/ABI its fine to use it here

Reply:
Okay, thanks for your inputs.

Reply:
Suggestion: Could we rename it to cfs_get_fs_id ( ) or cfs_fs_id_get( ) ?